### ISSUE TITLE
To do list

### ISSUE THREAD

**Issue by ChoiDM** [Yonsei-TAIL/KHD2020#2]
- [ ] **Network** (ResNet, EfficientNet, [ACM module](https://github.com/mk-minchul/attend-and-compare) 등)
- [ ] Pre-processing : **Resize**, Normalization, Pixel intensity windowing
- [ ] Optimization : Optimizer (Adam, SGD), **Learning rate**
- [ ] Loss function : Cross entropy, Focal loss (**class imbalance**)
